src/runtime-rs/crates/runtimes/virt_container/src/container_manager/manager.rs

```diff
@@ -272,4 +277,107 @@ impl ContainerManager for VirtContainerManager {
         process.process_type == ProcessType::Container
             && process.container_id.container_id == self.sid
     }
+
+    // if guest_swap is true, add swap to memory_sandbox
+    // returns `(memory_sandbox_b, swap_sandbox_b)` (in Byte)
+    async fn mem_required(&self) -> Result<(u64, i64)> {
```

Comment:
Could you leave this on another pr? As this is for memory

Reply:
I will leave this as a dummy one.